### PR TITLE
MacOS Exporter Dependency Bug

### PR DESCRIPTION
**Pull request by KyroVibe**
### Description
The Exporter was failing to install pip package dependencies on MacOS.

### Problem
The command to execute the Fusion supplied Python runtime had spaces in it and wasn't wrapped in quotations. I have no idea why this wasn't a problem before.

### Solution
Added quotations around failing install and uninstall commands.

### CODE REVIEW

**Comment by BrandonPacewic** [comment]
I have a few things I want to try out before we merge this one if that's alright. I'll try to be timely about it.

**Comment by KyroVibe** [comment]
> I have a few things I want to try out before we merge this one if that's alright. I'll try to be timely about it.

Sure thing

**Comment by KyroVibe** [comment]
@BrandonPacewic Sure thing. What do you want to try out?

**Comment by BrandonPacewic** [comment]
> @BrandonPacewic Sure thing. What do you want to try out?

Trying to fix the subprocess call.

**Comment by KyroVibe** [comment]
@BrandonPacewic Good luck with that

**Review by BrandonPacewic — APPROVED** [review]
I see what you mean, seems a little sub-optimal but what can you do 🤷‍♂️

**Pull request merged**
